BlueApp.js

```diff
@@ -12,7 +14,11 @@ const BlueApp = new AppStorage();
 
 async function startAndDecrypt(retry) {
   if (BlueApp.getWallets().length > 0) {
-    console.log('App already has some wallets, so we are in already started state, exiting startAndDecrypt');
+    Sentry.addBreadcrumb({
```

Comment:
Wouldn't it be better to have another level of abstraction here to cover `Sentry.addBreadcrumb()` and simplify usage of the logger? This way, instead of
```
  Sentry.addBreadcrumb({
      category: 'BlueApp',
      message: `App already has some wallets, so we are in already started state, exiting startAndDecrypt`,
      level: Sentry.Severity.Info,
    });
```

there could be something like:

```
  logger.info('BlueApp', `App already has some wallets, so we are in already started state, exiting startAndDecrypt`)
```

which says more for people who don't know what Sentry is.

Just a suggestion.

Reply:
good idea, I'll add it

Reply:
done